knx-linux/CMakeLists.txt

```diff
@@ -38,3 +77,5 @@ target_link_libraries(knx-linux  "${LIBRARIES_FROM_REFERENCES}")
 include_directories(../src)
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -Wall -Wno-unknown-pragmas -Wno-switch -g -O0")
 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wall -Wno-unknown-pragmas -Wno-switch -g -O0")
+set_property(TARGET knx-linux PROPERTY CXX_STANDARD 11)
+install(TARGETS knx-linux RUNTIME DESTINATION /tmp)
```

Comment:
I have just added this so that Qt Creator (my IDE) "knows" where to find the binary for deploying to the target while developing on my x86 Linux host machine. As I am running this as a normal user the usual paths /usr/local/bin, etc. won't work. So it is really a temporary file only while developing. Do you have any idea how to solve this?
BTW: Qt creator is also the reason why I have added all header files so that they show up in the project tree.

Reply:
I don't use Qt Creator, but for commandline you can set INSTALL_PREFIX or DESTDIR to change the prefix from /usr/local to something else like "/home/foobar". Qt Creator should have some option to set this.